OpenLaTeXStudio/Editor/pom.xml

```diff
@@ -98,6 +99,11 @@
         <dependency>
             <groupId>org.netbeans.api</groupId>
             <artifactId>org-openide-util</artifactId>
+            <version>${netbeans.version}</version>
```

Comment:
The versions should be declared in the parent pom.xml in the dependency management section